.taskcluster.yml

```diff
@@ -452,3 +452,40 @@ tasks:
           description: Deploy frontend build on environment
           owner: bastien@mozilla.com
           source: https://github.com/mozilla/code-coverage
+
+    - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+      then:
+        $let:
+          version: {$eval: 'head_branch[10:]'}
+        in:
+          taskId: {$eval: as_slugid("release")}
+          dependencies:
+            - {$eval: as_slugid("addon_build")}
+            - {$eval: as_slugid("bot_build")}
+            - {$eval: as_slugid("backend_build")}
+            - {$eval: as_slugid("frontend_build")}
+            - {$eval: as_slugid("report_check_tests")}
```

Comment:
Nit: We could use the deploy tasks as dependencies, so we don't release unless everything is fully green.

Reply:
That won't work as the events are not the same.
The workflow should become: 
1. push a tag to build a release (and support pre-release as RC to push tests)
2. push the tag on testing/production & use the previously built images

The advantage is that you build less, and use "human" tags in TC hooks. But that's quite some work

Reply:
Ah right, here the situation is a bit different than the other repos. We deploy on push to testing/production, we release on tag. Forget about this then.